mw-webapp/src/component/image/Image.tsx

```diff
@@ -25,21 +27,55 @@ interface ImageProps {
    * Data attribute for cypress testing
    */
   dataCy?: string;
+
+  /**
+   * Enlarge Image
+   */
+  enlargeImage?: boolean;
+
+  /**
+   * Enlarge Image Styles
+   */
+  enlargeClassName?: string;
 }
 
 /**
  * Сomponent for displaying images
  */
 export const Image = (props: ImageProps) => {
   const className = clsx(styles.image, props.className);
+  const enlargeClassName = clsx(styles.image, props.enlargeClassName);
 
   return (
-    <img
-      src={props.src}
-      alt={props.alt}
-      className={className}
-      data-cy={props.dataCy}
-    />
+    props.enlargeImage
```

Comment:
1. Let's create internal state to handle this instead of prop + external state
2. We should be able to disable zoomable behavior through props

Reply:
Done!

Reply:
I mean Image component dont need to have ability to become big programmatically, let's just handle click event